app/components/organisms/Sidebar.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box, Button, Image, Heading, HStack, Flex } from '@chakra-ui/react'
+import type { Puzzle } from '@prisma/client'
+import Form from '../atoms/Form'
+import SidebarMenu from '../molecules/SidebarMenu'
+
+export type SidebarProps = {
+  puzzles: Pick<Puzzle, 'id'>[]
+}
+
+const Sidebar = ({ puzzles }: SidebarProps) => {
+  return (
+    <Flex
+      h="full"
+      bg="background.dark"
+      as="aside"
+      alignItems="self-start"
+      px={4}
+      py={4}
+      flexDirection="column"
+      color="white"
+      gap={8}
+    >
+      <HStack>
+        <Image maxW="12" src="/images/chest.png" alt="logo app" />
```

Comment:
C'est une image d'illustration qui ne transmet pas d'information, il vaut mieux la masquer
```suggestion
        <Image maxW="12" src="/images/chest.png" aria-hidden="true" />
```